### PR TITLE
feat: [#178212559] Added learn more CTA on OutcomeCodeMessageComponent

### PR DESCRIPTION
**Pull request by andreafavaro-bitrocketdev**
## Short description
Added new CTA to show instructions on how to activate the 3DS on my card if the bank authorization fails

## How to test
1. Find this function "handlePaymentPostAndRedirect" on io-dev-api-server, change outcomeValue to 2, this will simulate a failed payment due to lack of authorization by the bank.
2. Try to pay a notice, the payment request will fail and  you should be able to see the button with the new CTA "Learn More"

NOTE: I added a "FIXME" comment in the code because the "Learn more" CTA points to an external site, yet to be defined.

**Before:**
Screenshot 2021-05-20 at 13 08 56

**After:**
Screenshot 2021-05-20 at 12 46 02

### CODE REVIEW

**Comment by thisisjp** [comment]
That's great! I'll get back to you once the webpage will be ready ;)

**Comment by Undermaken** [comment]
> That's great! I'll get back to you once the webpage will be ready ;)

@thisisjp any updates?

**Comment by Undermaken** [comment]
@thisisjp @debiff 
can we close this PR?

**Comment by thisisjp** [comment]
@Undermaken nope, please hold on ;)

**Comment by Undermaken** [comment]
@thisisjp is this PR obsolete now?

**Comment by thisisjp** [comment]
still valid, please hold on

**Comment by thisisjp** [comment]
Update: we'll soon have a valid URL once ticket [WEB-209] will be completed

File: ts/screens/wallet/payment/PaymentOutcomeCodeMessage.tsx [hunk 1]
```diff
@@ -64,6 +64,9 @@ const successFooter = (onClose: () => void) => (
 const PaymentOutcomeCodeMessage: React.FC<Props> = (props: Props) => {
   const outcomeCode = props.outcomeCode.outcomeCode.fold(undefined, oC => oC);
 
+  // FIXME: this CTA will point to an external site, yet to be defined
+  const onLearnMore = () => null;
```
Comment (thisisjp) [inline ts/screens/wallet/payment/PaymentOutcomeCodeMessage.tsx:68]: @Undermaken the final URL is https://io.italia.it/faq/#pagamenti

Comment (thisisjp) [inline ts/screens/wallet/payment/PaymentOutcomeCodeMessage.tsx:68]: @Undermaken the final URL is https://io.italia.it/faq/#pagamenti

Comment (debiff) [reply ts/screens/wallet/payment/PaymentOutcomeCodeMessage.tsx]: Updated in [01c15ac](https://github.com/pagopa/io-app/pull/3057/commits/01c15acd8a56958e36a7da56cfde21d8918861f7)
https://user-images.githubusercontent.com/11773070/163563996-c16425b2-1605-4fb7-a2f0-407bdf158600.mp4

**Pull request merged**